src/Symfony/Bridge/Monolog/Handler/FingersCrossed/HttpCodeActivationStrategy.php

```diff
@@ -15,10 +15,14 @@
 use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\HttpKernel\Exception\HttpException;
 
+@trigger_error(sprintf('Since symfony/monolog-bridge 5.1: The "%s" class is deprecated, use "%s" instead.', HttpCodeActivationStrategy::class, HttpCodeActivationStrategyDecorator::class), E_USER_DEPRECATED);
```

Comment:
```suggestion
trigger_deprecation('symfony/monolog-bridge', '5.1', 'The "%s" class is deprecated, use "%s" instead.', HttpCodeActivationStrategy::class, HttpCodeActivationStrategyDecorator::class);
```

We should use the new deprecation contracts here.

Reply:
Actually I tried but tests were failing with PHP 7.4, I don't know why. The change is done in a fixup commit so I can revert it easily

Reply:
done